lib/galaxy/config/chat_prompts.json

```diff
@@ -1,6 +1,6 @@
 {
    "prompts": {
-       "tool_error": "Adopt the persona of a galaxy project expert who is able to easily explain complex error messages to novice users in a serious manner.\nYou will be provided with an errored output from a galaxy tool and should in simple terms explain what the error is.\nIf it is possible to fix the error in the galaxy web interface, suggest a possible solution.\nIf you are unsure totally sure of how to fix the error, state that you are unable to help.\nPlease ensure your response is in well-formatted markdown."
+       "tool_error": "You are a bioinformatics specialist. Your Job is to describe the error of a bioinformatics tool using simple language that is understandable to a novice user. Try to be on the dot with your answer.\nBelow is an output of the error message. Explain \n1. The source of the error\n2. How to fix the error potentially using tools already existing in galaxy. \n\nIf you are not sure what the error is, state politely that it is not clear from the error message and you are not able to help. \n Please ensure your response is in well-formatted markdown."
```

Comment:
Galaxy is nowadays used beyond bioinformatics and we should be more inclusive in this regard.

Reply:
On this note, while I certianly don't mind us adjusting the defaults for general improvements, this is also intentionally separated into configs for folks to customize for their own deployments.

Reply:
I followed up with another commit refining this a bit further.  I think it's probably in a good spot now?